src/main/resources/db/changelog/db.changelog-1.0.xml

```diff
@@ -0,0 +1,24 @@
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <changeSet id="1" author="authorName">
+        <createTable tableName="mail_message_info">
+            <column name="id" type="bigint" autoIncrement="true">
+                <constraints primaryKey="true"  nullable="false"/>
```

Comment:
I did same mistake:)
Autoincrement cannot be null especially for bigint. So you can remove 'nullable=false'

Reply:
fixed